reactome_analysis_api/reactome_analysis_api/searcher/overview_fetcher.py

```diff
@@ -0,0 +1,68 @@
+import grein_loader
+import requests
+import json
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Fetcher():
```

Comment:
This is just a point to discuss: Maybe add this code to the searcher class as the concept is quite similar?

Reply:
The public data searcher is already a bit full, I think for better maintenance and better abstraction we should separate the data fetching and the logic of the searching.